client/components/plans/plans-compare/index.jsx

```diff
@@ -117,15 +112,15 @@ var PlansCompare = React.createClass( {
 		}
 	},
 
-	isSubmitting: function() {
+	isSubmitting() {
 		if ( ! this.props.transaction ) {
 			return false;
 		}
 
-		return this.props.transaction.step.name === transactionStepTypes.SUBMITTING_WPCOM_REQUEST
+		return this.props.transaction.step.name === SUBMITTING_WPCOM_REQUEST
```

Comment:
@Tug made the suggestion that we import the whole module, and then use the full reference, like `transactionStepTypes.SUBMITTING_WPCOM_REQUEST`, to make it clearer in the code what that constant refers to. We lose that with this change.